hw/ip/aes/doc/_index.md

```diff
@@ -133,14 +139,25 @@ The AES unit operates as follows:
 1. The initial key and configuration is provided to the AES unit via a set of control and status registers (CSRs) accessible by the processor via TL-UL bus interface.
    Each key register has to be written at least once.
    The order in which the registers are written does not matter.
+1. The processor provides the initialization vector (IV) or initial counter value to the four IV registers via TL-UL bus interface in CBC or CTR mode, respectively.
+   Each IV register has to be written at least once.
+   The order in which the registers are written does not matter.
+   In ECB mode, no IV must be provided.
```

Comment:
^must be^need be^ perhaps? Because otherwise you'd have to do something
like alert if it was written, and that seems not worth it.

Reply:
Yes, I agree. I changed it.